### PR TITLE
feat(dropdown): add support for dropup

### PR DESCRIPTION
**Pull request by jnizet**
also improves the demo to show how to align the dropdown menu to the right
capture d ecran 2016-09-02 a 19 46 30

### CODE REVIEW

File: src/dropdown/dropdown.ts [hunk 1]
```diff
@@ -7,14 +7,20 @@ import {Directive, Input, Output, HostListener, EventEmitter} from '@angular/cor
...
+    '[class.dropdown]': '!up',
+    '[class.dropup]': 'up',
     '[class.open]': 'isOpen()',
     '(keyup.esc)': 'closeFromOutside()',
     '(document:click)': 'closeFromOutside()'
   }
 })
 export class NgbDropdown {
   /**
+   * Indicates that dropdown should in fact drop up
```
Comment (pkozlowski-opensource) [inline src/dropdown/dropdown.ts]: There is a "be" missing somewhere here, no?

Comment (icfantv) [inline src/dropdown/dropdown.ts]: I would say "Indicates that the dropdown should open upwards" or something similar.

**Comment by icfantv** [comment]
Just my one comment and i can fix the language on the merge.

**Comment by jnizet** [comment]
I made the change. Thanks for the review.

**Pull request closed without merging**